wicker/core/storage.py

```diff
@@ -52,36 +75,41 @@ def __init__(self) -> None:
         """Constructor for a file system storage instance."""
         pass
 
-    @retry(
-        Exception,
-        tries=get_config().storage_download_config.retries,
-        backoff=get_config().storage_download_config.retry_backoff,
-        delay=get_config().storage_download_config.retry_delay_s,
-        logger=logger,
-    )
-    def download_with_retries(self, input_path: str, local_path: str):
-        try:
-            shutil.copyfile(input_path, local_path)
-        except Exception as e:
-            logging.error(f"Failed to download/move object for file path: {input_path}")
-            raise e
-
```

Comment:
Before, `fetch_file` used to call this `download_with_retries` function above. Instead, let's remove the `download_with_retries` function that has the unnecessary `@retry` decorator and just change `fetch_file` to call the `shutil.copyfile(input_path, local_path)` function directly instead.